AISKU/src/Initialization.ts

```diff
@@ -189,6 +189,14 @@ export class Initialization implements IApplicationInsights {
             _core = new AppInsightsCore();
             _self.core = _core;
 
+        
+            if (!config.connectionString) {
+                _throwInternal(_core.logger,
+                    eLoggingSeverity.CRITICAL,
+                    _eInternalMessageId.InstrumentationKeyDeprecation,
+                    "Instrumentation key based ingestion support will end, please use connection string instead. More information at: https://docs.microsoft.com/en-us/azure/azure-monitor/app/migrate-from-instrumentation-keys-to-connection-strings");
```

Comment:
Can we create an aka.ms alias for this as it will be a LOT shorter as strings can't be minified, so this is a lot of extra text.

Mabe something like
"Instrumentation key support will end soon, see: aka.ms/xxxxxx"